<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserControlPanel</class>
 <widget class="QDialog" name="UserControlPanel">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="homeTab">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>636</width>
      <height>453</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>636</width>
      <height>453</height>
     </size>
    </property>
    <attribute name="title">
     <string>Home</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item row="9" column="0" colspan="3">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="loginCancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loginLoginButton">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="loginPassField">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="loginPassValid">
       <property name="styleSheet">
        <string notr="true">color:#F00;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="loginEmailValid">
       <property name="styleSheet">
        <string notr="true">color:#F00;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="loginEmailLabel">
       <property name="text">
        <string>email</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>User Login</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="loginEmailField"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="loginPassLabel">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="scoreTab">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="title">
     <string>Score</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="registerTab">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>636</width>
      <height>453</height>
     </size>
    </property>
    <attribute name="title">
     <string>Register</string>
    </attribute>
    <widget class="QLabel" name="regUserLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
     <property name="buddy">
      <cstring>regUserField</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="regUserField">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>180</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="regPassField2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>362</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="regLNameLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>119</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
     <property name="buddy">
      <cstring>regLNameField</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="regFNameLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>59</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
     <property name="buddy">
      <cstring>regFNameField</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="regLNameField">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>119</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="regPassLabel2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>362</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm Pass</string>
     </property>
     <property name="buddy">
      <cstring>regPassField2</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="regPassValid">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>334</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F00;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>regPassField</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="regFNameValid">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>91</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F00;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>regFNameField</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="regPassValid2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>394</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F00;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>regPassField2</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="regLNameValid">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>151</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F00;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>regLNameField</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="regUserValid">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>212</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F00;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>regUserField</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="regEmailValid">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>273</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#F00;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>regEmailField</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="regTitleLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>411</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Registration Form</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="regPassLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>302</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="buddy">
      <cstring>regPassField</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="regPassField">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>302</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="regEmailLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>241</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
     <property name="buddy">
      <cstring>regEmailField</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="regEmailField">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>241</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="regFNameField">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>59</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="regRegisterButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>410</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="regCancelButton">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>410</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>loginEmailField</tabstop>
  <tabstop>loginPassField</tabstop>
  <tabstop>loginLoginButton</tabstop>
  <tabstop>loginCancelButton</tabstop>
  <tabstop>regFNameField</tabstop>
  <tabstop>regLNameField</tabstop>
  <tabstop>regUserField</tabstop>
  <tabstop>regEmailField</tabstop>
  <tabstop>regPassField</tabstop>
  <tabstop>regPassField2</tabstop>
  <tabstop>regRegisterButton</tabstop>
  <tabstop>regCancelButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
